CHANGELOG.rst

```diff
@@ -3,6 +3,7 @@
 
 Added
 ^^^^^
+* `@j-rausch`_: Added unit tests for changed lingual parsing pipeline
```

Comment:
Thanks! Leave these sentences ending in a period `.`. Also, you'll have to add the ref to your name at the bottom of the file. For example:
```
.. _@j-rausch: https://github.com/j-rausch
```

You can test that it looks correct locally by running `make docs` and opening `_build/html/index.html` in a browser.